src/contribute/contribution-guide.md

```diff
@@ -244,16 +244,28 @@ All code should have appropriate unit testing coverage. New code should have
 new tests in the same contribution. Bug fixes should include a regression test
 to prevent the issue from reoccurring.
 
-#### Java SDK
+The entire set of unit tests can be run with this command at the root of the
+git repository.
 
-For contributions to the Java code, run unit tests locally via Maven.
+    $ ./gradlew test
```

Comment:
Perhaps adding some instructions for calling unit tests on certain languages may be useful. This is slightly different than the precommit info down below because it is only unit tests.

Examples:
```
$ ./gradlew sdks:java:test
$ ./gradlew sdks:python:test
$ ./gradlew sdks:go:test
```

Also may be worth using the "check" task instead of "test". According to the gradle docs:

> It is common for all verification tasks, including tests and linting, to be executed using the check task. ([Source](https://docs.gradle.org/current/userguide/command_line_interface.html#_running_all_checks))

I don't know if Beam is doing this at the moment, but I feel like this area of the code should mention using check for more intense checks (i.e. for replacing mvn verify), while test is better for basic unit tests.

Reply:
Replaced with check, which includes lint (on some SDKs).
I replaced the precommit tasks below with *:check, for language SDKs that had them (I don't know if there's a Java check task for all parts of the Java SDK).